scripts/nix-build.in

```diff
@@ -56,7 +56,7 @@ if ($runEnv && defined $ARGV[0] && $ARGV[0] !~ /nix-shell/) {
             @ARGV = ();
             while (<SCRIPT>) {
                 chomp;
-                if (/^\#\!\s*nix-shell (.*)$/) {
+                if (/^.*\s*nix-shell (.*)$/) {
```

Comment:
You're right. Actually `.*\s*` is equivalent to just `.*`. Having `\s*` doesn't bring anything to the table since it's part of the `.*` set.